modules/vpc/main.tf

```diff
@@ -1340,3 +1340,163 @@ resource "aws_s3_bucket_object" "public_state" {
+                ]
+            },
+            "Action": [
+                "sts:AssumeRole"
+            ]
+        }
+    ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "user_management" {
+    count = "${var.enabled * var.enable_user_management * length(split(",", var.environments))}"
+
+    lifecycle {
+        create_before_destroy = true
+    }
+
+    name = "user_management-${var.aws_region}-${element(split(",", var.environments), count.index)}"
+    role = "${element(aws_iam_role.user_management.*.id, count.index)}"
+    policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "LambdaVPCAccess",
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+                "ec2:CreateNetworkInterface",
+                "ec2:DescribeNetworkInterfaces",
+                "ec2:DeleteNetworkInterface"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "CreateIAMUsers",
+            "Effect": "Allow",
+            "Action": [
+                "iam:List*",
+                "iam:Get*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOF
+}
+
+resource "aws_lambda_function" "user_management" {
+    count = "${var.enabled * var.enable_user_management * length(split(",", var.environments))}"
+
+    depends_on = [
+        "aws_iam_role_policy.user_management"
+    ]
+
+    function_name   = "user_management-${element(split(",",var.environments), count.index)}"
+    s3_bucket       = "nubis-stacks"
+    s3_key          = "${var.nubis_version}/lambda/UserManagement.zip"
+    role            = "${element(aws_iam_role.user_management.*.arn, count.index)}"
+    handler         = "index.handler"
+    description     = "Queries LDAP and inserts user into consul and create and delete IAM users"
+    memory_size     = 128
+    runtime         = "nodejs4.3"
+    timeout         = "30"
+
+    vpc_config = {
+        subnet_ids = [
+            "${element(aws_subnet.private.*.id, 3*count.index)}",
+            "${element(aws_subnet.private.*.id, 3*count.index+1)}",
+            "${element(aws_subnet.private.*.id, 3*count.index+2)}",
+        ]
+        security_group_ids = [
+            "${element(aws_security_group.shared_services.*.id, count.index)}",
+            "${element(aws_security_group.internet_access.*.id, count.index)}",
+            "${element(aws_security_group.ldap.*.id, count.index)}",
+        ]
+    }
+}
+
+resource "aws_security_group" "ldap" {
+    count = "${var.enabled * length(split(",", var.environments))}"
+
+    lifecycle {
+        create_before_destroy = true
+    }
+
+    vpc_id      = "${element(aws_vpc.nubis.*.id, count.index)}"
+    name_prefix = "MocoLdapOutbound-${element(split(",", var.environments), count.index)}-"
+    description = "Allow outbound ldap connection to moco ldap"
+
+    egress {
```

Comment:
First, is this egress rule even strictly speaking necessary?
Second, you are apparently hard-coding a Mozilla IP address in there, how could we avoid that?